apps/teams-test-app/src/components/OtherAppStateChangeAPIs.tsx

```diff
@@ -0,0 +1,53 @@
+import { otherAppStateChange } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+import { ApiWithoutInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const CheckOtherAppStateChangeCapability = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'otherAppStateChange_isSupported',
+    title: 'Check OtherAppStateChanged Capability',
+    onClick: async () => `OtherAppStateChanged module ${otherAppStateChange.isSupported() ? 'is' : 'is not'} supported`,
+  });
+
+const RegisterAppInstallHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'otherAppStateChange_registerInstallHandler',
+    title: 'Register App Install Handler',
+    onClick: async (setResult) => {
+      const handler = (event: otherAppStateChange.OtherAppStateChangeEvent): void => {
+        console.log(event);
```

Comment:
Is this line on purpose to console the log?

Reply:
It was because it seemed useful for debugging, but I can definitely remove it.